src/amo/css/ScreenShots.scss

```diff
@@ -1,6 +1,8 @@
 @import "~core/css/inc/mixins";
 @import "~amo/css/inc/vars";
-$screenshot-height: 200px;
+
+$screenshot-height: 220px;
+$image-height: $screenshot-height - 20px;
```

Comment:
could you please add comment here so its clear why we're doing this -  maybe describe the issue and add a reference to the issue